core/sector_storage/stores/index.hpp

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_SECTOR_INDEX_HPP
+#define CPP_FILECOIN_CORE_SECTOR_INDEX_HPP
+
+#include "common/outcome.hpp"
+#include "primitives/sector/sector.hpp"
+#include "primitives/sector_file/sector_file.hpp"
+#include <chrono>
+
+namespace fc::sector_storage::stores {
+  // ID identifies sector storage by UUID. One sector storage should map to one
+  //  filesystem, local or networked / shared by multiple machines
+  using ID = std::string;
+  using fc::primitives::sector::RegisteredProof;
+  using fc::primitives::sector::SectorId;
+  using fc::primitives::sector_file::SectorFileType;
+  using std::chrono::system_clock;
+
+  const std::chrono::seconds kHeartbeatInterval(10);
+  const std::chrono::seconds kSkippedHeartbeatThreshold =
+      kHeartbeatInterval * 5;
+
+  struct StorageInfo {
+    ID id;
+    std::vector<std::string> urls;  // TODO: Support non-http transports
```

Comment:
Not clear what changes are implied by the todo. Please create a task in jira with the desired changes description and leave the id here.

Reply:
Fixed